internal/app.go

```diff
@@ -114,7 +116,7 @@ func (a *App) extract(config configSnapshot) error {
 
 	all_fname := make(map[string]bool)
 	for fname, cfgFile := range config.next {
-		slog.Info("Extracting file", "file", fname)
+		a.InfoLogger("Extracting file", "name", fname)
```

Comment:
I wonder if we could configure slog in such the way that it modifies output format at the command initialization and does not require any changes in code that uses slog, so each slog.Info/Debug occurrences remain intact.

I believe @detsch investigated such possibility for zerolog, and answer was positive. Specifically, zerolog.Info would produce normal log like message in case of the daemon operation and terminal like output in case of the CLI commands. Now, we need to check it out for slog.

Reply:
I've force-pushed a new variation that is looking better to me.